.editorconfig

```diff
@@ -183,6 +183,9 @@ dotnet_diagnostic.CA1827.severity = warning
 # Use Length/Count property instead of Enumerable.Count method.
 dotnet_diagnostic.CA1829.severity = warning
 
+# Use span-based 'string.Concat' (incompatible with mono builds).
```

Comment:
Maybe at least add a TODO here, reminding this should be enabled if/when we drop Mono?

Reply:
mono is mentioned